README.md

```diff
@@ -25,6 +25,7 @@ The following table describes the properties of the `options` object.
 |---------------|-----------------------------------------|--------|-------------|
 | `name`        | name of the server timing metric        | String | 'mw'        |
 | `description` | description of the server timing metric | String | `undefined` |
+| `active`      | set `false` if you want to disable      | Boolean| `true`      |
```

Comment:
Thanks for the submission!!

Would you be willing to change this to a `disabled` flag instead? I personally think it makes it more obvious that it is truly optional. WDYT?

Reply:
Makes sense. I'll change it.